File: data/rename_casualties/BeforeMethodRename.cpp
class MyClass {
private:
    int x;
    int y;

    int getX() const {
        return x;
    }

    int getY() const {
        return y;
    }

public:
    MyClass() : x(0), y(0) {}

    int getSum() const {
        return getX() + getY();
    }
};

File: src/test/data/rename_casualties/BeforeMethodRename.java
public class MyClass {
    private int x, y;

    public MyClass() {
        this.x = 0;
        this.y = 0;
    }

    public int getSum() {
        return getX() + getY();
    }

    private int getX() {
        return this.x;
    }

    private int getY() {
        return this.y;
    }
}